package.json

```diff
@@ -77,5 +77,10 @@
 		"playwright": "^1.42.1",
 		"react-modal": "^3.16.1",
 		"react-tooltip": "^5.26.3"
+	},
+	"dependencies": {
+		"axios": "^1.7.2",
```

Comment:
Rather not introduce axios, let's use fetch instead.

Reply:
resolved! marking conversation as resolved at this time.